src/utils/config.py

```diff
@@ -31,11 +31,20 @@ def get_config():
         'GLOBAL_CONFIG_URL',
         'https://github.com/kbase/search_config/releases/latest/download/config.yaml'
     )
+    github_release_url = os.environ.get(
+        'GITHUB_RELEASE_URL',
+        'https://api.github.com/repos/kbase/search_config/releases/latest'
+    )
     # Load the global configuration release (non-environment specific, public config)
     if not config_url.startswith('http'):
         raise RuntimeError(f"Invalid global config url: {config_url}")
-    with urllib.request.urlopen(config_url) as res:  # nosec
-        global_config = yaml.safe_load(res)  # type: ignore
+    if not github_release_url.startswith('http'):
+        raise RuntimeError(f"Invalid global github release url: {github_release_url}")
+    try:
+        with urllib.request.urlopen(config_url) as res:  # nosec
+            global_config = yaml.safe_load(res)  # type: ignore
+    except Exception as err:
+        raise RuntimeError(f"config url I'm trying to use {config_url}, {github_release_url}")
```

Comment:
This is a weird error message. Can you make it say something like f"Error loading configuration from {config_url}""